hydra-tui/src/Hydra/Client.hs

```diff
@@ -95,7 +95,7 @@ withClient Options{hydraNodeHost = Host{hostname, port}, cardanoSigningKey, card
   handleDisconnect f =
     callback ClientDisconnected >> threadDelay 1 >> reconnect f
 
-  externalCommit' sk payload =
+  requestCommitTx' sk payload =
```

Comment:
Could: This is not only requesting the commit tx, but actually does signing and submission as well, so we could name it more different than "just requesting"

Reply:
This is a _victim_ of mass rename I did... reverting.